tests/models/test_match_spec.py

```diff
@@ -964,3 +964,18 @@ def test_md5_merge_wo_name(self):
         assert str(merged[0]) in str_specs
         assert str(merged[1]) in str_specs
         assert str(merged[0]) != str(merged[1])
+
+
+    def test_dist_str(self):
+        m1 = MatchSpec.from_dist_str("anaconda/{}::python-3.6.6-0.tar.bz2".format(context.subdir))
```

Comment:
When using `.format()` and positional arguments, best practice is to enumerate each position. So 

```
"anaconda/{0}::python-3.6.6-0.tar.bz2".format(context.subdir)
```

One reason is that it has a substantial speed advantage, approaching the speed of the older `%` string composition. The performance obviously doesn't matter here, but it can make a difference in a tight loop.